tests/gpts/bitbucket/configs/bitbucket_positive_suite.ts

```diff
@@ -0,0 +1,216 @@
+import { afterAll, beforeAll, describe, expect, it, jest } from '@jest/globals';
+import { DeveloperHubClient } from '../../../../src/apis/backstage/developer-hub';
+import { TaskIdReponse } from '../../../../src/apis/backstage/types';
+import { generateRandomChars } from '../../../../src/utils/generator';
+import { BitbucketProvider } from "../../../../src/apis/scm-providers/bitbucket";
+import { Kubernetes } from "../../../../src/apis/kubernetes/kube";
+import { checkEnvVariablesBitbucket, cleanAfterTestBitbucket, createTaskCreatorOptionsBitbucket, getDeveloperHubClient, geBitbucketClient, getRHTAPRootNamespace, checkIfAcsScanIsPass, verifySyftImagePath } from "../../../../src/utils/test.utils";
+
+/**
+ * 1. Components get created in Red Hat Developer Hub
+ * 2. Check that components gets created successfully in Red Hat Developer Hub
+ * 3. Red Hat Developer Hub created Bitbucket repository
+ * 4. Perform an commit in Bitbucket to trigger a push PipelineRun
+ * 5. Wait For PipelineRun to start and finish successfully. This is not done yet. We need SprayProxy in place and
+ * wait for RHTAP bug to be solved: https://issues.redhat.com/browse/RHTAPBUGS-1136
+ */
+export const bitbucketSoftwareTemplateTests = (gptTemplate: string) => {
+    describe(`Red Hat Trusted Application Pipeline ${gptTemplate} GPT tests Bitbucket provider with public/private image registry`, () => {
+        jest.retryTimes(2);
+
+        const componentRootNamespace = process.env.APPLICATION_ROOT_NAMESPACE || 'rhtap-app';
+        const developmentNamespace = `${componentRootNamespace}-development`;
+
+        const bitbucketUsername = process.env.BITBUCKET_USERNAME || '';
+        const bitbucketAppPassword = process.env.BITBUCKET_APP_PASSWORD || '';
```

Comment:
Is this var necessary for test suite? It is necessary only for BitBucket client, but you have geBitbucketClient, which takes value from RHTAP namespace